cartoframes/dataset.py

```diff
@@ -256,8 +267,11 @@ def _get_read_query(self, table_columns, limit=None):
         return query
 
     def get_table_columns(self):
-        """Get column names and types from a table"""
-        query = 'SELECT * FROM "{schema}"."{table}" limit 0'.format(table=self.table_name, schema=self.schema)
+        """Get column names and types from a table or a query result"""
```

Comment:
While looking at this method, I've realized the branch you want to merge to is not synced with master... I'd merge with master to avoid problems in the future...

Reply:
Yes, you are right, we have added commits to `master` after the `develop` creation. We should do it in the `develop` branch rather than here.

Reply:
Done